### ISSUE TITLE
Add support for syncing deeply nested Models and Collections

### ISSUE THREAD

**Issue by astoilkov** [astoilkov/jsblocks#76]
Currently if you call sync on Model or Collection that have deeply nested Model or Collection inside this will throw an error when `JSON.stringify` is executed.

**Comment by quantumlink** [comment]
Is there a way to implement syncing for now without this that you could recommend?

**Comment by astoilkov** [comment]
Yes. However, you will need to manually do your ajax requests and then manually fill the Model and Collection objects. You could try that and write back if you need more help with this.
